API/NoDocs/Microsoft.Extensions.Hosting.Testing/FakeHost.cs

```diff
@@ -0,0 +1,20 @@
+// Assembly 'Microsoft.Extensions.Hosting.Testing'
+
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Extensions.Hosting.Testing;
+
+public sealed class FakeHost : IHost, IDisposable
+{
+    public IServiceProvider Services { get; }
+    public static IHostBuilder CreateBuilder();
+    public static IHostBuilder CreateBuilder(Action<FakeHostOptions> configure);
+    public static IHostBuilder CreateBuilder(FakeHostOptions options);
```

Comment:
It's strange to have both patterns. It's easy to get from one to the other. WebApplication.CreateBuilder uses the direct options approach.
```suggestion
    public static IHostBuilder CreateBuilder(FakeHostOptions options);
```

Reply:
Out of time, keeping this as-is.